build-system/tasks/visual-diff/snippets/freeze-form-values.js

```diff
@@ -0,0 +1,47 @@
+// This file is executed via Puppeteer's page.evaluate on a document to remove
+// all <script> tags that import AMP pages. This makes for cleaner diffs and
+// prevents "double-execution" of AMP scripts when enableJavaScript=true.
```

Comment:
Nice. Double execution was an annoying problem when we used `enableJavaScript=true` a long time ago. Does this PR re-enable that option?

Reply:
Oops, this comment comes from the file `remove-amp-scripts.js`, which I had copied to create this file. Pushed an updated comment.

The fix for `enableJavaScript=true` has been in for a while now :)